manifests/attribute_resolver/attribute.pp

```diff
@@ -10,11 +10,16 @@
   $source_attribute_id = undef,
   $transient = false,
   $script = undef,
+  $script_file = undef,
 
   $dependencies = {},
   $encoders = {},
 ) {
 
+  if $type == 'Script' {
+    fail("Attribute type 'Script' must be replaced with 'ScriptedAttribute' as of Shibboleth IdP 3.3")
```

Comment:
This release should clearly state, in the changelog, it is for 3.3 and above.

Reply:
I updated the release and CHANGELOG.